source/renderer/app/components/wallet/settings/ICOPublicKeyDialog.tsx

```diff
@@ -131,14 +130,12 @@ class ICOPublicKeyDialog extends Component<Props> {
         actions={actions}
         closeOnOverlayClick
         onClose={onClose}
-        className={styles.dialog}
         closeButton={<DialogCloseButton onClose={onClose} />}
       >
         <div className={styles.description}>
           {intl.formatMessage(messages.description)}
         </div>
         <Input
-          className={styles.spendingPassword}
```

Comment:
I guess/hope that these are stale class names that do not exist in the CSS anymore?

Reply:
I just double checked on develop branch and there's no `spendingPassword` in the `source/renderer/app/components/wallet/settings/PublicKeyDialog.scss`